Sources/ArcGISToolkit/Extensions/ArcGIS/AttachmentHelpers/FeatureAttachment.swift

```diff
@@ -64,6 +64,18 @@ public protocol FeatureAttachment: Loadable {
     ///   - height: Height of the thumbnail.
     /// - Returns: A task that represents the asynchronous operation. The task result contains the thumbnail as an `UIImage`.
     func makeThumbnail(width: Int, height: Int) async throws -> UIImage
+    
+    /// Loads the metadata for the instance asynchronously.
+    ///
+    /// A workaround providing conforming types an opportunity to call their load method directly.
+    /// - Bug: Apollo #1058
+    func _load() async throws
+    
+    /// The load status.
+    ///
+    /// A workaround providing conforming types an opportunity to access their load status directly.
+    /// - Bug: Apollo #1058
+    var _loadStatus: LoadStatus { get }
```

Comment:
Adding these allows the two conforming types (`FormAttachment` and `PopupAttachment`), to call their `load()` and `loadStatus` members directly. This is the cleanest workaround I've found so far.